### ISSUE TITLE
Incorrect capitalisation in images

### ISSUE THREAD

**Issue by samgrieve** [tianevitt/tianzes-otari-extras#8]
There is incorrect capitalisation on the name of the module folder on most of the images for maps etc which causes blank maps until each tile or token is edited.

**Comment by tianevitt** [comment]
Thank you. This issue was not evident even on my separate test machine, but I will go through and do a consistency check.

**Comment by samgrieve** [comment]
I run two Foundry instances cloud hosted on unix servers. Had the same issue with both servers when adding the module. Going through the removing the capitalisation from every image sorted it. Will probably only be an issue for those hosting on unix servers.

**Comment by tianevitt** [comment]
Yes, I figured you were on some flavor of a unix server, which I neglected to think of when assembling all the assets. This issue probably impacts Macs as well.

Yikes at all the effort you went through, because I just went through the same thing myself, and am about to upload what I hope will fix everything. I have no way of testing it for certain, because I don't have a unix-like option at the moment. I have been working on standing up an oracle cloud umbuntu instance, which will hopefully be done in the next day or so.
